### PR TITLE
Add documentation for distillation column model

### PR DESCRIPTION
**Pull request by AlbertLee125**
This pull request adds documentation for the distillation column model. It includes documentation for the `build_model` function in `column.py`, the `initialize` function in `initialize.py`, and the `calculate_Fenske` function. Additionally, it fixes a grammar error in `column.py` and fills in missing documentation. The documentation provides details on the purpose and parameters of each function, making it easier for developers to understand and use the code.

### CODE REVIEW

File: gdplib/gdp_col/column.py [hunk 1]
```diff
@@ -8,10 +8,28 @@
...
+    -------
+    Pyomo.ConcreteModel
+        A Pyomo model of the distillation column for separation of benzene and toluene.
+    """
     m = ConcreteModel('benzene-toluene column')
-    m.comps = Set(initialize=['benzene', 'toluene'])
-    min_T, max_T = 300, 400
+    m.comps = Set(initialize=['benzene', 'toluene'], doc='Set of components')
+    min_T, max_T = 300, 400 # Define temperature bounds [K]
     max_flow = 500
```
Comment (bernalde) [inline gdplib/gdp_col/column.py]: Missing units

**Review by bernalde — APPROVED** [review]
Only one small comment on missing flow units but after that it would be good to merge!

File: gdplib/gdp_col/column.py (hunk 1 above)
Comment (AlbertLee125) [reply gdplib/gdp_col/column.py]: added the missing units and fixed the typo in the description.

**Comment by bernalde** [comment]
@ZedongPeng I don;t know if the linter is working well, but then should we block this until it passes the test?

**Comment by AlbertLee125** [comment]
@ZedongPeng It seems that I have to do the documentation in double quotes(" ") instead of single quote (' ').

**Comment by ZedongPeng** [comment]
I have added the `-S -C` arguments, so the black format check will ignore the quotes. Let me check it.

> @ZedongPeng It seems that I have to do the documentation in double quotes(" ") instead of single quote (' ').

**Comment by ZedongPeng** [comment]
> @ZedongPeng I don;t know if the linter is working well, but then should we block this until it passes the test?

Let's use the PR to test and make the linter work as expected. I will commit to this branch.

**Comment by ZedongPeng** [comment]
The quotes issue is resolved, but it seems that many files (not in the distillation column) do not satisfy the black format.
I think we can merge this PR first and I will create a new PR to format the whole repository. @bernalde 
(I didn't set any rule related to Lint check, so this PR is ready to be merged.)

**Review by ZedongPeng — APPROVED** [review]

**Pull request merged**